docs/ops/detection/PriorBoxClustered_1.md

```diff
@@ -6,105 +6,102 @@
 
 **Short description**: *PriorBoxClustered* operation generates prior boxes of specified sizes normalized to the input image size.
 
+**Detailed description**
+
+*PriorBoxClustered* computes coordinates of prior boxes by following:
+1.  Calculates the *center_x* and *center_y* of prior box:
+    \f[
+    W \equiv Width \quad Of \quad Image
+    \f]
+    \f[
+    H \equiv Height \quad Of \quad Image
+    \f]
+    \f[
+    center_x=(w+offset)*step
+    \f]
+    \f[
+    center_y=(h+offset)*step
+    \f]
+    \f[
+    w \subset \left( 0, W \right )
+    \f]
+    \f[
+    h \subset \left( 0, H \right )
+    \f]
+2.  For each \f$s \subset \left( 0, W \right )\f$ calculates the prior boxes coordinates:
+    \f[
+    xmin = \frac{center_x - \frac{width_s}{2}}{W}
+    \f]
+    \f[
+    ymin = \frac{center_y - \frac{height_s}{2}}{H}
+    \f]
+    \f[
+    xmax = \frac{center_x - \frac{width_s}{2}}{W}
+    \f]
+    \f[
+    ymax = \frac{center_y - \frac{height_s}{2}}{H}
+    \f]
+If *clip* is defined, the coordinates of prior boxes are recalculated with the formula:
+\f$coordinate = \min(\max(coordinate,0), 1)\f$
+
 **Attributes**
 
 * *width (height)*
```

Comment:
Maybe we should have separate items for *width* and *height*?

Reply:
I'm strongly against separating items for nearly identical attributes, which seems like it could inflate the spec size, making it harder to read, with no apparent benefit from it.